core/src/main/scala/com/mesosphere/usi/core/Scheduler.scala

```diff
@@ -43,9 +44,14 @@ import scala.concurrent.Future
  *                                     +----------------------+
  */
 object Scheduler {
+
+  private val log = Logger.takingImplicit[KvArgs](getClass)
+
   case class MesosConnection(mesosHostName: String)
 
   def connect(mesosHostName: String): Future[(MesosConnection, Flow[SpecEvent, StateEvent, NotUsed])] = {
+    val composedLogger = log.withCtx("mesosHostName", mesosHostName)
+    composedLogger.info(s"Connecting to $mesosHostName")(composedLogger.ctx)
```

Comment:
Why do we have to pass `composedLogger.ctx`? Shouldn't the logger have its context already?